core-lib/Files.ns

```diff
@@ -516,11 +544,25 @@ DAMAGE.>> *)
       ^ self pattern
     )
 
-    public open: mode <Symbol> ifFail: err <[:IOException | X def]>  ^ <FileDescriptor | X> = (
+    (* Open file with given access mode.
+       Access mode can be #read, #write, or #readWrite. *)
+    public open: mode <Symbol> ifFail: errBlock <[:Symbol | X def]> ^ <FileDescriptor | X> = (
       (* Modes: #read, #write, #readWrite *)
```

Comment:
i guess thats no longer needed?

Reply:
* the comment line

Reply:
Why? Think it is good practice to provide comments to all methods.
Ideally, VS code is going to show them at some point. (Think we only need to track them during parsing, if we don't do it yet, and then show them on the corresponding request. should be pretty simple)

Reply:
i mean the old one, the content is in the new one, so redundant

Reply:
aaah, now I see it, yes :) thanks